docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,161 @@
+
+
+## I Have a Question
+
+> If you want to ask a question, we assume that you have read the available [Wiki](https://github.com/GeorgeCiesinski/poke-guesser-bot/wiki).
+
+Before you ask a question, it is best to search for existing [Issues](https://github.com/GeorgeCiesinski/poke-guesser-bot/issues) that might help you. In case you have found a suitable issue and still need clarification, you can write your question in this issue. It is also advisable to search the internet for answers first.
+
+If you then still feel the need to ask a question and need clarification, we recommend the following:
+
+- Open an [Issue](https://github.com/GeorgeCiesinski/poke-guesser-bot/issues/new).
+- Provide as much context as you can about what you're running into.
+- Provide project and platform versions (nodejs, npm, etc), depending on what seems relevant.
+
+We will then take care of the issue as soon as possible.
+
+> Alternatively, you can join our [Discord](#community) and ask your question there.
+
+
+## I Want To Contribute
+
+> ### Legal Notice
+> When contributing to this project, you must agree that you have authored 100% of the content, that you have the necessary rights to the content and that the content you contribute may be provided under the project license.
+
+### Reporting Bugs
+
+#### Before Submitting a Bug Report
+
+A good bug report shouldn't leave others needing to chase you up for more information. Therefore, we ask you to investigate carefully, collect information and describe the issue in detail in your report. Please complete the following steps in advance to help us fix any potential bug as fast as possible.
+
+- Make sure that you are using the latest version.
+- Determine if your bug is really a bug and not an error on your side e.g. using incompatible environment components/versions (Make sure that you have read the [Wiki](https://github.com/GeorgeCiesinski/poke-guesser-bot/wiki). If you are looking for support, you might want to check [this section](#i-have-a-question)).
+- To see if other users have experienced (and potentially already solved) the same issue you are having, check if there is not already a bug report existing for your bug or error in the [bug tracker](https://github.com/GeorgeCiesinski/poke-guesser-bot/labels/bug).
+- Also make sure to search the internet (including Stack Overflow) to see if users outside of the GitHub community have discussed the issue.
+- Collect information about the bug:
+- Stack trace (Traceback) if applicable
+- OS, Platform and Version (Windows, Linux, macOS, x86, ARM)
+- Version of the interpreter, compiler, SDK, runtime environment, package manager, depending on what seems relevant.
+- Steps to reproduce the bug
+
+
+#### How Do I Submit a Good Bug Report?
+
+> For security related issues, please report a security vulnerability [here](https://github.com/GeorgeCiesinski/poke-guesser-bot/security/advisories/new).
```

Comment:
You'd need to create one for that link to work, I think:
![image](https://user-images.githubusercontent.com/30803034/221386872-6c182f1e-92e4-45a9-bb1d-12fa89ecf098.png)

When you go on https://github.com/GeorgeCiesinski/poke-guesser-bot/security/advisories it seems like that there are the settings for it.

Reply:
I was trying to link to this issue type. Do you know if contributors have access to this? Do you have access?

<img width="1257" alt="Screenshot 2023-02-26 at 3 38 52 PM" src="https://user-images.githubusercontent.com/7014211/221435928-f44479a6-2679-4eee-89ad-0ddba777423d.png">

Reply:
This is in the issues tab: https://github.com/GeorgeCiesinski/poke-guesser-bot/issues

Reply:
I also double checked and it looks like it is for publicly reporting security issues, so it should work: 

https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability

Reply:
I first checked the advisories link than I got this:
![image](https://user-images.githubusercontent.com/30803034/221436162-7a187fae-7579-4550-ab5d-edbe1ed98b13.png)
Then I clicked on New on Issues and got:
![image](https://user-images.githubusercontent.com/30803034/221436216-0a0b1031-a159-4b77-8059-958ac5079d71.png)
After that I've then closed the tab and reopened it with Ctrl+Shift+T and now I see the third option:
![image](https://user-images.githubusercontent.com/30803034/221436197-79da5b9e-125a-4f9e-a47b-e10a2c84d0a4.png)

So TL;DR I seem to see it now, don't know if non Collaborators see it.

Reply:
I think I fixed it, which is why it worked for you. I had to enable a beta option in the settings for it. Okay then in that case I will update it.

Reply:
Jip, now the link with the new in the end seems to work fine, I don't get a 404 anymore